README.md

```diff
@@ -169,45 +172,45 @@ Testing base64 encoding/decoding of the large blob into the newly allocated buff
-|                   Node.js |  1.743<sub>±0.006</sub> |   40.76<sub>±00.04</sub> + 38.86<sub>±00.02</sub> |  70.15<sub>±00.65</sub> |
-|       Perl (MIME::Base64) |  1.791<sub>±0.000</sub> |    14.70<sub>±00.08</sub> + 0.16<sub>±00.03</sub> |  70.22<sub>±00.07</sub> |
-|                        Go |  1.875<sub>±0.002</sub> |     4.22<sub>±00.05</sub> + 3.92<sub>±00.05</sub> |  80.99<sub>±00.49</sub> |
-|                       PHP |  2.093<sub>±0.017</sub> |    17.79<sub>±00.03</sub> + 0.00<sub>±00.00</sub> |  84.17<sub>±00.78</sub> |
-|                     D/gdc |  2.442<sub>±0.003</sub> |     7.52<sub>±00.07</sub> + 3.35<sub>±00.00</sub> | 108.08<sub>±00.52</sub> |
-|                     D/dmd |  2.739<sub>±0.001</sub> |     3.60<sub>±00.12</sub> + 3.85<sub>±00.05</sub> | 119.70<sub>±00.68</sub> |
-|                    Python |  2.837<sub>±0.002</sub> |    10.36<sub>±00.03</sub> + 0.55<sub>±00.06</sub> | 112.86<sub>±00.30</sub> |
-|                       Zig |  3.067<sub>±0.002</sub> |     1.45<sub>±00.04</sub> + 0.00<sub>±00.00</sub> | 122.43<sub>±00.62</sub> |
-|              F#/.NET Core |  3.202<sub>±0.052</sub> |   38.62<sub>±00.06</sub> + 18.68<sub>±00.71</sub> | 120.67<sub>±01.23</sub> |
-|               Python/pypy |  3.581<sub>±0.002</sub> |   64.67<sub>±00.11</sub> + 31.23<sub>±00.04</sub> | 156.54<sub>±00.94</sub> |
-|                       Tcl |  3.685<sub>±0.078</sub> |     5.12<sub>±00.06</sub> + 0.00<sub>±00.00</sub> | 149.06<sub>±01.75</sub> |
-|                  Go/gccgo |  3.720<sub>±0.002</sub> |    25.62<sub>±00.20</sub> + 7.71<sub>±00.17</sub> | 168.84<sub>±00.90</sub> |
-|              C#/.NET Core |  3.726<sub>±0.053</sub> |   34.67<sub>±00.08</sub> + 19.67<sub>±01.47</sub> | 137.24<sub>±01.33</sub> |
-|                   C#/Mono |  4.692<sub>±0.014</sub> |   25.71<sub>±00.08</sub> + 18.74<sub>±00.03</sub> | 193.29<sub>±01.07</sub> |
-|                     Julia |  5.023<sub>±0.004</sub> |  244.56<sub>±00.09</sub> + 42.28<sub>±00.09</sub> | 188.22<sub>±02.10</sub> |
-|  Ruby/truffleruby (--jvm) |  5.072<sub>±0.012</sub> | 347.27<sub>±07.42</sub> + 168.12<sub>±27.78</sub> | 234.49<sub>±01.53</sub> |
-|                Ruby/jruby | 10.519<sub>±0.026</sub> |  182.81<sub>±02.29</sub> + 80.63<sub>±02.65</sub> | 420.63<sub>±01.57</sub> |
-| Perl (MIME::Base64::Perl) | 13.790<sub>±0.053</sub> |    16.12<sub>±00.07</sub> + 0.31<sub>±00.04</sub> | 573.08<sub>±04.60</sub> |
-|          Ruby/truffleruby | 16.232<sub>±0.006</sub> | 295.77<sub>±00.95</sub> + 232.04<sub>±04.13</sub> | 647.25<sub>±03.84</sub> |
+|            C/gcc (aklomp) |  0.099<sub>±0.000</sub> |     2.07<sub>±00.02</sub> + 0.00<sub>±00.00</sub> |   4.64<sub>±00.06</sub> |
+|          C/clang (aklomp) |  0.099<sub>±0.001</sub> |     2.07<sub>±00.04</sub> + 0.00<sub>±00.00</sub> |   4.59<sub>±00.07</sub> |
+|                       PHP |  0.106<sub>±0.000</sub> |    17.31<sub>±00.09</sub> + 0.00<sub>±00.00</sub> |   4.78<sub>±00.07</sub> |
```

Comment:
php what? they implement aklomp algorithm?

Reply:
Hm, `V` has `-prealloc` argument which turns on arena allocation. Other languages have such flags perhaps too. Shouldn't that be considered here if C++ and Zig have already gotten this "privilege"?

Ping @spytheman .